.goreleaser.yml

```diff
@@ -67,26 +67,26 @@ dockers:
   - kando
   image_templates:
   - 'kanisterio/es-sidecar:{{ .Tag }}'
-  dockerfile: 'examples/helm/kanister/kanister-elasticsearch/image/Dockerfile'
+  dockerfile: 'docker/kanister-elasticsearch/image/Dockerfile'
   extra_files:
-  - 'examples/helm/kanister/kanister-elasticsearch/image/esdump-setup.sh'
+  - 'docker/kanister-elasticsearch/image/esdump-setup.sh'
 - binaries:
   - kando
   image_templates:
   - 'kanisterio/mysql-sidecar:{{ .Tag }}'
-  dockerfile: 'examples/helm/kanister/kanister-mysql/image/Dockerfile'
+  dockerfile: 'docker/kanister-mysql/image/Dockerfile'
```

Comment:
I was wondering if all the `Dockerfile`s that we are moving from `examples` to `docker` are actually being used somewhere.

Reply:
So, the purpose was to move the docker files that are being used in releaser to `docker` dir, so yes, they are being used